app/javascript/app/data/ignored-countries.js

```diff
@@ -0,0 +1,2 @@
+// We won't show Taiwan and Western Sahara as an independent country
+export const IGNORED_COUNTRIES_ISOS = ['TWN', 'ESH'];
```

Comment:
This could actually live in constants.js but it's ok, it's more tragic this way.

Reply:
Yes, maybe having a separate file is a bit too much :) I was thinking where to put this const, but `constant.js` seems to be very "crowded" to me, we should organize the constants in a more logical way